fedimint-sqlite/src/lib.rs

```diff
@@ -136,6 +136,34 @@ impl<'a> IDatabaseTransaction<'a> for SqliteDbTransaction<'a> {
         Box::pin(stream::iter(rows))
     }
 
+    async fn raw_find_by_prefix_sorted_descending(
+        &mut self,
+        key_prefix: &[u8],
+    ) -> Result<PrefixStream<'_>> {
+        let mut str_prefix = "".to_string();
+        for prefix in key_prefix {
+            str_prefix = format!("{str_prefix}{prefix:02X?}");
```

Comment:
This seems like a meh way to format this prefix allocating new string for every byte.

```
format!("{}%", to_hex(key_prefix));
```

kind of thing should work instead?

Reply:
Yeah, it seems inneficient

Reply:
Replaced by `hex::encode`